src/AWS.Deploy.Constants/ElasticBeanstalk.cs

```diff
@@ -22,7 +22,7 @@ internal static class ElasticBeanstalk
         public const string HealthCheckURLOptionNameSpace = "aws:elasticbeanstalk:application";
         public const string HealthCheckURLOptionName = "Application Healthcheck URL";
 
-        public const string LinuxPlatformType = ".NET Core";
+        public const string LinuxPlatformType = ".NET";
```

Comment:
Can we add a trailing space, `".NET "`,  to help avoid getting false positives like `.NETWORK`. Unlikely to happen but figure it doesn't hurt to add.

Reply:
Done via https://github.com/aws/aws-dotnet-deploy/pull/807/commits/7ad351b8373436f48d05971784a763e67ac7a504